server/src/server.ts

```diff
@@ -122,3 +122,15 @@ connection.onDocumentFormatting(
 documents.listen(connection);
 
 connection.listen();
+
+function findNumberOfLines (textDocument: string): number{
+	let count = 1;  // Line number
+	for (let i = 0; i < textDocument.length; i++){
+		if (textDocument[i] === '\n'){
+			count ++; 
+		}
+	}
+
+	console.log (count); 
+	return count; 
+}
```

Comment:
There is not API to get the size of the file?

Reply:
'documentText' here is only a string. If we want to get the line number, we need some variable directly related to the document.